modules/ROOT/pages/examples/small-scale.adoc

```diff
@@ -229,6 +232,65 @@ Reload nginx again:
 sudo systemctl reload nginx
 ----
 
+If the server is accessible from the web, see the free {ssl-labs-url}[SSL Labs] page to check the sites SSL security. 
+
 Open a web browser, navigate to your Infinite Scale URL `\https://{ocis_url}` and log in as admin user with the password returned by the `ocis init` command.
 
+== Apache as Reverse Proxy
+
+Always run a configuration test when updating your Apache configuration with:
+
+[source,bash]
+----
+sudo apachectl configtest
+----
+
+=== Prepare Apache for certbot
+
+To set up Apache as a reverse proxy with Let's Encrypt TLS certificates, create a file called e.g. `ocis` under `/etc/apache2/sites-available` with the following content:
+
+include::partial$deployment/apache_80_example.adoc[]
+
+Activate the new Apache config and reload Apache:
+
+[source,bash]
+----
+sudo ln -s /etc/apache2/sites-available/ocis /etc/apache2/sites-enabled/ocis
+----
+
+[source,bash]
+----
+sudo systemctl reload apache2
+----
+
+=== Issuing a Certificate via certbot for Apache
+
+Run `certbot` to issue the TLS certificates:
+
+[source,bash,subs="attributes+"]
+----
+sudo certbot --apache -d {ocis_url}
+----
+
+=== Finalize the Apache Configuration
+
+Adapt the config to redirect automatically to https, use the newly generated certificates and add the required proxy configuration. The full config should look like this:
+
+include::partial$deployment/apache_full_example.adoc[]
+
+Add the following to the SSL definition above if SSL security should be hardened / improved:
+
+include::partial$deployment/apache_harden_ssl.adoc[]
+
+Reload Apache:
+
+[source,bash]
+----
+sudo systemctl reload nginx
```

Comment:
This looks wrong - it should be reloading/restarting Apache

Reply:
fixed